sklearn/neighbors/quad_tree.pyx

```diff
@@ -279,7 +275,7 @@ cdef class _QuadTree:
         cdef bint res = True
         for i in range(self.n_dimensions):
             # Use EPSILON to avoid numerical error that would overgrow the tree
-            res &= fabsf(point1[i] - point2[i]) <= EPSILON
+            res &= fabs(point1[i] - point2[i]) <= EPSILON
```

Comment:
we're using double not instead of float? is there no fabsf in ``libc.math``?

Reply:
My mistake. `fabsf` is in `math.h` but for some reason not re-exported in cython's `libc.math` I think. Reverted that particular change.